h/browser/chrome/lib/sidebar-injector.js

```diff
@@ -12,11 +11,12 @@ function toIIFEString(fn) {
   return '(' + fn.toString() + ')()';
 }
 
-function addMetaTagFn(name, content) {
-  var metaTag = document.createElement('meta');
-  metaTag.name = name;
-  metaTag.content = content;
-  document.head.appendChild(metaTag);
+function extractContentScriptResult(result) {
```

Comment:
A comment here linking to https://developer.chrome.com/extensions/tabs#method-executeScript and explaining (briefly) why the result is an array would be good.